composer/loggers/progress_bar_logger.py

```diff
@@ -175,11 +175,10 @@ def log_traces(self, traces: Dict[str, Any]):
                 self._log_to_console(f'[trace]: {trace_name}:' + trace_str + '\n')
 
     def log_hyperparameters(self, hyperparameters: Dict[str, Any]):
-        if self.should_log_to_console:
```

Comment:
why is if statement removed?

Reply:
It enables hyperparameter logging for the case where console logging is off and only progress bar logging is on. Will enable the desired behavior from the Jira example (Hagay used just progress bar with no console logging).